cms/standard_pages/tests/factories.py

```diff
@@ -0,0 +1,47 @@
+from datetime import timedelta
+from typing import ClassVar
+
+import factory
+import wagtail_factories
+from django.utils import timezone
+
+from cms.core.tests.factories import ContactDetailsFactory, RichTextBlockFactory
+from cms.standard_pages.models import MethodologyPage
+from cms.topics.tests.factories import TopicPageFactory
+
+
+class SectionContentBlockFactory(wagtail_factories.StructBlockFactory):
+    """Factory for Section content block."""
+
+    title = factory.Faker("text", max_nb_chars=50)
+    content = wagtail_factories.StreamFieldFactory(
+        {
+            "rich_text": factory.SubFactory(RichTextBlockFactory),
+        }
+    )
+
+
+class SectionBlockFactory(wagtail_factories.StructBlockFactory):
+    """Factory for Section block."""
+
+    title = factory.Faker("text", max_nb_chars=50)
+    content = factory.SubFactory(SectionContentBlockFactory)
```

Comment:
this are probably better suited placed in `cms.core.tests.factories` and given they are a copy of the factories from `cms.analysis.tests.factories`, can you tidy that one too? (that is remove from there and import them from `cms.core.tests.factories`

Reply:
Done [539a1d3](https://github.com/ONSdigital/dis-wagtail/pull/57/commits/539a1d3cddbf3cb8b37c875d29d6208f457a1e0b)

I've replied with a question to another comment about this issue below so I think this one can be closed since the core issue has been solved and we can keep the conversation under the other comment.